opsdroid/core.py

```diff
@@ -205,7 +205,18 @@ def start_databases(self, databases):
                 _LOGGER.debug("Processing parsers...")
                 parsers = self.config["parsers"]
 
-                dialogflow = [p for p in parsers if p["name"] == "dialogflow"]
+                dialogflow = [p for p in parsers if p["name"] == "dialogflow"
+                              or p["name"] == "apiai"]
+
+                # Show deprecation message but  parse message
+                # Once it stops working remove this bit
+                # pylint: disable= using-constant-test
+                if "apiai":
```

Comment:
Jacob could you let me know if this statement is acceptable? Pylint was giving me a using-constant-test error so I disabled it. I tried to write this a few different ways but couldn't get the logging message to show. 

I've also added some comments so we can delete this bit of code when apiai stops working, I hope this is okay?

Reply:
I'm not sure that line makes sense. You're basically checking if a string is `True`.

I would recommend you restore the line:

```
apiai = [p for p in parsers if p["name"] == "apiai"]
```

And then change your if to say:

```
if apiai:
    _LOGGER.warning("Api.ai is now called Dialogflow. This "
                     "parser will stop working in the future "
                     "please swap: 'name: apiai' for "
                     "'name: dialogflow' in configuration.yaml")
```

You wouldn't use the `apiai` variable for anything else.